cabal.project

```diff
@@ -0,0 +1,5 @@
+packages: bytestring.cabal
+allow-newer: *:bytestring
+
+tests: False
+benchmarks: False
```

Comment:
Why is this necessary when there aren't any testsuites or benchmarks?

Reply:
Probably redundant, but harmless. Belt and suspers, but the belt is quite strong...